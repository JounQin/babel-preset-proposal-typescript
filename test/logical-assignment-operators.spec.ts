import { case1, case2, case3 } from './logical-assignment-operators'
import { execute } from './helpers'

const proposal = 'logical-assignment-operators'

test(proposal, () => {
  expect(() => execute(proposal)).toThrowErrorMatchingSnapshot()
  expect(case1(0, 2)).toEqual([0, 2])
  expect(case1(2, 0)).toEqual([2, 2])
  expect(case2(0, 2)).toEqual([0, 0])
  expect(case2(2, 0)).toEqual([2, 0])
  expect(case3(0, 2)).toEqual([0, 2])
  expect(case3(2, 0)).toEqual([2, 0])
})
