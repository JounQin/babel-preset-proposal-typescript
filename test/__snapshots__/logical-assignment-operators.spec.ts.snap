// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`logical-assignment-operators 1`] = `"Unexpected token )"`;
